### DISCUSSION TITLE
backup feature/attribute

### DISCUSSION THREAD

**Discussion by geir75** [foldynl/QLog#558] [Ideas]
Dear developers,
I've been using qlog for some time now and I'm really fine with it. What I maybe miss is some kind of efficient backup/restore feature/attribute. Like we have a backup/restore feature/attribute in tqsl. I believe making it as a feature/attribute in the software is more efficient and secure. I presume location of the backup/restore feature/attribute can be set to be under Logbook menu.

**Comment by foldynl** [comment]
Under Linux/MacOS missing backup is not an issue. If you back up your HOME directory then everything QLog needs will be backed up as well because everything is located in your HOME directory. Additionally QLog automatically creates a log snapshots every 7 days. Everything is described on the [WIKI](https://github.com/foldynl/QLog/wiki/Autobackup).

The situation is a bit different under Windows. Under Windows everything except configuration parameters is stored in the USER directory. Unfortunately configuration parameters are stored in the Registry. This means you could lose local settings but not your QSOs if you make a backup regularly

I believe that applications themselves should not handle backup/restore functionality. This should be managed by a separate application. However, what QLog lacks and what has been on the TODO list for a long time is the ability to migrate from one computer to another.

**Comment by TecnoTalarn** [comment]
Hello
if I want to copy my configuration and data from the installation via PPA in ubuntu to flatpack, how do I do it?
I see that the flatpack version works with hamlib 4.6 and that helps me a lot with my FT-710, but the installation in flatpck does not see any data from the ppa.
This is the first time I use flatpack, and I have no idea how it works, or where the data is.
I managed to pass QLog,conf, but it does not solve my equipment configuration, or my profile.
I also did not find the contact database, although I could solve this with an ADIF backup. 

How can it be solved?

thanks

**Comment by foldynl** [comment]
copy all files

from `$HOME/.local/share/hamradio/QLog` to `$HOME/.var/app/io.github.foldynl.QLog/data/hamradio/QLog`

and copy the config file

`$HOME/.config/hamradio/QLog.conf` to `$HOME/.var/app/io.github.foldynl.QLog/config/hamradio/QLog.conf`

**Reply by TecnoTalarn** [reply]
Thanks a lot!
